spec/lib/search/pagination_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Search::Pagination do
+  [nil, 20, 40, 60, 80].each do |next_offset|
+    context "when next_offset is #{next_offset}" do
+      let(:raw_body) do
+        {
+          'total' => 84,
+          'next_offset' => next_offset
+        }
+      end
+      subject { described_class.new(raw_body) }
+
+      it 'calculates the correct previous offset' do
+        prev_offset = case next_offset
+                      when 20 # Cursor on first page
+                        nil
+                      when 40 # Cursor on second page
+                        nil
```

Comment:
Pls see:

> If the user is on the 2nd page of results, and they want to get the 1st page, in this case we'd want previous: 0 (not previous: nil), so that we are consistent with how the FE requests the 1st page, vs. indicating to the FE that they currently have the first page. You know what I mean?